src/System.Data.SqlClient/tests/ManualTests/SQL/SqlConnectionTest/SqlConnectionTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Data.SqlClient.ManualTesting.Tests;
+using System.Linq;
+using System.Security;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public static class SqlConnectionTest
```

Comment:
Could you drop the test logins at the end of the tests? Ideally as part of a try-finally statement.